app/flatpak-builtins-build-export.c

```diff
@@ -652,7 +653,7 @@ flatpak_builtin_build_export (int argc, char **argv, GCancellable *cancellable,
   guint64 installed_size = 0,download_size = 0;
   GTimeVal ts;
 
-  context = g_option_context_new (_("LOCATION DIRECTORY [BRANCH] - Create a repository from a build directory"));
+  context = g_option_context_new (_("LOCATION DIRECTORY [[COLLECTION-ID] BRANCH] - Create a repository from a build directory"));
```

Comment:
Not sure about the order here. You might want to have a branch, but not set a collection id, whereas there is always a branchname, even if you leave the argument out.

In fact, isn't it better if this is --collection-id=ID